docs/pages/foundations/messaging/priority_and_placement.js

```diff
@@ -0,0 +1,326 @@
+              </Box>
+              <Box padding={4} width="50%" display="none" lgDisplay="block">
+                <Text weight="bold">Example wireframe</Text>
+              </Box>
+            </Flex>
+            <Box color="errorWeak">
+              <Flex direction="column">
+                <Box color="errorBase" height={4} width="100%" />
+                <Box display="flex" alignItems="center" direction="column" lgDirection="row">
+                  <Box padding={4} width="50%">
+                    <Text>
+                      To acknowledge completion of a task that doesn’t require follow up, like
+                      pinning to a board. Use{' '}
+                      <Link href="/web/toast" underline="always" inline>
+                        Toast
+                      </Link>{' '}
+                      instead.
+                    </Text>
+                  </Box>
+                  <Box padding={4} width="50%">
+                    <PinSaved />
+                  </Box>
+                </Box>
+                <Divider />
+                <Box display="flex" alignItems="center" direction="column" lgDirection="row">
+                  <Box padding={4} width="50%">
+                    <Text>
+                      To provide general information and recommendations. Use{' '}
+                      <Link href="/web/callout" underline="always" inline>
+                        Callout
+                      </Link>{' '}
+                      instead.
+                    </Text>
+                  </Box>
+                  <Box padding={4} width="50%">
+                    <NewRecommendations />
+                  </Box>
+                </Box>
+                <Divider />
+                <Box display="flex" alignItems="center" direction="column" lgDirection="row">
+                  <Box padding={4} width="50%">
+                    <Text>
+                      To show an error or warning that applies to a section of a page, or that
+                      doesn’t severely limit the user experience.
+                    </Text>
+                  </Box>
+                  <Box padding={4} width="50%">
+                    <CampaignIssue />
+                  </Box>
+                </Box>
+              </Flex>
+            </Box>
+          </Box>
+        </MainSection.Subsection>
+        <MainSection.Subsection title="Not-blocking">
+          <Heading size="300" accessibilityLevel={4}>
+            Ephemeral—less prominent
+          </Heading>
+          <Emphemeral />
+          <Text>
+            Small overlays that appear for a short period of time, usually 3–5 seconds depending on
+            content. They do not prevent a user from accessing the main content and navigation on
+            the surface underneath.
+          </Text>
+          <Text>
+            Current components:{' '}
+            <Link href="/web/toast" underline="always">
+              Toast
+            </Link>
+          </Text>
```

Comment:
This "Current components" piece isn't consistent across the page. I like the way it's done for Modal, maybe it should be bold everywhere?

Reply:
Crap - I missed this part. Fixing now.

Reply:
Done!